applications/ShallowWaterApplication/tests/shallow_water_test_factory.py

```diff
@@ -3,15 +3,11 @@
 # Importing the Kratos Library
 import KratosMultiphysics
 import KratosMultiphysics.ShallowWaterApplication
-try:
-    import KratosMultiphysics.ExternalSolversApplication
-    have_external_solvers = True
-except ImportError as e:
-    have_external_solvers = False
+import KratosMultiphysics.ExternalSolversApplication
```

Comment:
theoretically the linear-sovler-factory should import this

Reply:
removed